src/egon/data/datasets/low_flex_scenario/__init__.py

```diff
@@ -15,16 +15,16 @@ class LowFlexScenario(Dataset):
     def __init__(self, dependencies):
         super().__init__(
             name="low_flex_scenario",
-            version="0.0.1",
+            version="0.0.2",
             dependencies=dependencies,
             tasks=(
                 {
-                    PostgresOperator(
+                    SQLExecuteQueryOperator(
                         task_id="low_flex_eGon2035",
                         sql=files(__name__)
                         .joinpath("low_flex_eGon2035.sql")
                         .read_text(encoding="utf-8"),
-                        postgres_conn_id="egon_data",
+                        conn_id="egon_data",
```

Comment:
I remember that I just changed this lately for release 1.0.0. Did the behavior change again?

Reply:
Okay, thanks for the explanation!